sys/auto_init/auto_init.c

```diff
@@ -243,6 +243,11 @@ void auto_init(void)
         test_utils_interactive_sync();
     }
 
+    if (IS_USED(MODULE_DUMMY_THREAD)) {
+        extern void dummy_thread_create(void);
+        dummy_thread_create();
+    }
```

Comment:
Why not move this all the way to the top, right before the event thread init. This way it will skew all the thread ID's by one when enabled, triggering even more test failures :smiling_imp: